### DISCUSSION TITLE
GPU Instanced trees have no wind

### DISCUSSION THREAD

**Discussion by FrodoUndead** [visualdesigncafe/nature-renderer#40] [Help]
I recently upgraded and purchased Nature Renderer 2021 because of its ability to render trees. In order to make this work I had to upgrade my project and upon importing and updating my prefab's shaders with the included tool, I found that the wind was no longer working on my trees. Wind on the grass seems to be working fine however. 

I made sure to apply all updates that were available and re-import the asset folder. I also tried with multiple different shaders to see if it was specific to the shader I was using. It seems that nothing makes any changes and any shader that I upgrade with has the same issue. Wind also still works on the Terrain with the Nature Renderer shader if I have the script disabled and render through Unity instead.

I'm not really sure how to proceed since I haven't been able to find any documentation on wind so any help would be greatly appreciated.

**Comment by maxplooi** [comment]
What shaders are you using for the wind on the trees? It may be either an issue with the shader patcher, or a compatibility issue with the shader. If you can let me know the shaders then it will be easier to find out.

**Reply by FrodoUndead** [reply]
I tried using the standard SpeedTree8 shaders as well as #NVJOB stc8 shaders. Both of these shaders use a single image for both the leaves and the trunk, so I'm not sure if that's the issue. I also tried changing the prefabs to Static, and then changing the material to GPU instanced. 

Any help would be appreciated.
